banners/mobile/content/BannerTextVar.vue

```diff
@@ -0,0 +1,36 @@
+<template>
+	<div class="wmde-banner-message">
+		<div>
+			<p class="banner-text-title">
+				<strong>Hi,</strong>
+			</p>
+			<p>
+				vielleicht kommen wir gerade ungelegen, aber dennoch: Klicke jetzt bitte nicht weg! Am heutigen
+				{{ currentDayName }}, den {{ currentDate }}, bitten wir dich bescheiden, die Unabhängigkeit von
+				Wikipedia zu unterstützen. Insgesamt spenden 99% nichts - sie übergehen diesen Aufruf. Die
+				durchschnittliche Spende beträgt 22,25&nbsp;€, doch bereits 10&nbsp;€ helfen uns weiter.
+				{{ visitorsVsDonorsSentence }} Die meisten Menschen spenden, weil sie Wikipedia nützlich finden.
+				Hat Wikipedia dir in diesem Jahr Wissen im Wert einer Tasse Kaffee geschenkt? Dann entscheide dich,
+				eine der seltenen Ausnahmen zu sein, und gib etwas zurück. Vielen Dank!
+			</p>
+		</div>
+	</div>
+</template>
+
+<script setup lang="ts">
+import { computed, inject } from 'vue';
+import { DynamicContent } from '@src/utils/DynamicContent/DynamicContent';
+
+const {
+	currentDayName,
+	visitorsVsDonorsSentence
+}: DynamicContent = inject( 'dynamicCampaignText' );
+
+const currentDate = computed( () => {
```

Comment:
I think you don't need this as the dunamicCampaignText will provide it. Eg:

```js
const {
	currentDayName,
    currentDate,
	visitorsVsDonorsSentence
}: DynamicContent = inject( 'dynamicCampaignText' );
```

Reply:
Done.

Reply:
Can you also do the same in the Slides components?

Reply:
Done.